build_tools/setup_helpers/extension.py

```diff
@@ -19,6 +19,9 @@
 
 
 def _get_build(var):
+    if (var == "BUILD_KALDI") and not os.environ.get(var):
+        return True
+
```

Comment:
Although this works for our purpose, let's keep this function generic. How about this?

```python
def _get_build(var, default_val=False):
    if val not in os.ennviron:
        return default_val
    val = os.environ.get(var, default_val)
    ...
```

Reply:
good point 👍